server/embed/serve.go

```diff
@@ -224,6 +224,7 @@ func configureHttpServer(srv *http.Server, cfg config.ServerConfig) error {
 	// todo (ahrtr): should we support configuring other parameters in the future as well?
 	return http2.ConfigureServer(srv, &http2.Server{
 		MaxConcurrentStreams: cfg.MaxConcurrentStreams,
+		NewWriteScheduler:    http2.NewRandomWriteScheduler,
```

Comment:
Don't understand. `http2.NewRandomWriteScheduler` is different than `http2.NewPriorityWriteScheduler`

Reply:
Sorry for unclear comment.

For the `MaxConcurrentStreams`, the user can use `--max-concurrent-streams` to config etcd http2 server.
So I was thinking if the pr can support to use config to choose scheduler writer, as it is behavior change.

> If the response size is less than max frame size ([by default 1MiB][1]), the priority writer is not too bad in low traffic scenario.

[1]: https://cs.opensource.google/go/x/net/+/refs/tags/v0.8.0:http2/http2.go;l=70

Reply:
LIFO is completely not compatible with existence of watches (that are at the end of the handling queue). 
I think it's better to avoid having too many flags that are difficult to understand. 
This setting changed so many times in live-time of 3.4.x, that If any depended on the exact semantic, we would learn about it earlier.